docs/pages/guides/prebuild.md

```diff
@@ -0,0 +1,168 @@
+
+We plan to support more platforms in the future, but team size and limited resources prevent us from being able to maintain them at the moment. The stability of out-of-tree platforms like `react-native-macos` and `react-native-windows` are also considered.
+
+You can still use prebuild for your `ios`, and `android` projects but any extra platforms will have to be configured manually in the meantime.
+
+### Standard native development is slower
+
+All native changes must be added with Expo Modules Core and Config Plugins. This means if you want to add some quick native file to your project the old fashion way, then your better off adding the file manually and working your way back into the system via a monorepo (we recommend [expo-yarn-workspaces](https://www.npmjs.com/package/expo-yarn-workspaces)). We plan to speed this process up by adding functionality to Expo Autolinking that finds project native files outside of the native folders and links them before building.
+
+If you want to do something like modifying the gradle properties file, you'll have to write a plugin for that [example](https://github.com/expo/expo/blob/1c994bb042ad47fbf6878e3b5793d4545f2d1208/apps/native-component-list/app.config.js#L21-L28). Of course this could be easily automated with helper plugin libraries, but it is a bit slower if you need to do it often.
+
+### Config Plugin support
+
+Not all packages support _Expo Prebuilding_ yet. If you find a library that requires extra setup after installation and doesn't have an Expo Config Plugin, you should open a PR or an issue so that the maintainer is aware of the feature request.
+
+Some packages like [`react-native-blurhash`](https://github.com/mrousavy/react-native-blurhash) don't require any additional native setup so they don't even need a Config Plugin!
+
+Other packages like [`react-native-ble-plx`](https://github.com/Polidea/react-native-ble-plx) do require additional setup and therefore require a Config Plugin to be used with `npx expo prebuild` (in this case there's an external plugin called [`@config-plugins/react-native-ble-plx`](https://github.com/expo/config-plugins/tree/main/packages/react-native-ble-plx)).
+
+Alternatively, we also have a repo for [out-of-tree Expo Config Plugins][config-plugins-repo] which provides plugins for popular packages that haven't adopted the system yet. Think of this like [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) for TypeScript. We prefer packages ship their own Expo Config Plugin to ensure versioning is always aligned, but if they haven't adopted the system yet, the community can easily get unblocked.
+
+[config-plugins-repo]: https://github.com/expo/config-plugins
+[template]: https://github.com/expo/expo/tree/main/templates/expo-template-bare-minimum
+[native-modules]: /workflow/glossary-of-terms/#native-module
+[eas]: /eas
+[expo-go]: https://expo.dev/expo-go
+[config-plugins]: /guides/config-plugins/
+[expo-config]: /workflow/configuration/
```

Comment:
This could be a list of resources

```suggestion
- [config-plugins-repo]: https://github.com/expo/config-plugins
- [template]: https://github.com/expo/expo/tree/main/templates/expo-template-bare-minimum
- [native-modules]: /workflow/glossary-of-terms/#native-module
- [eas]: /eas
- [expo-go]: https://expo.dev/expo-go
- [config-plugins]: /guides/config-plugins/
- [expo-config]: /workflow/configuration/
```

Reply:
Will they still work as shared links? `[foo][eas]`

Reply:
No they won't (just tested that locally). Also, prior to your comment, I didn't know and realize they were shared links. This is so cool! TIL!

Please ignore & dismiss this suggestion 🙏